java/Oleksii-Riabov/src/com/zoolatech/lecture6/tasks/_1/Task1.java

```diff
@@ -0,0 +1,60 @@
+package com.zoolatech.lecture6.tasks._1;
+
+import java.util.*;
```

Comment:
try to not use `*` import

Reply:
Wildcard imports have been changed on explicit imports.